README.md

```diff
@@ -0,0 +1,35 @@
+# school-library
+this project is ruby application for school library management.
+
+## Built with
+Ruby
+
+## Getting started
+Clone the repos in your local machine.   
+Run bundle install to get the dependencies.  
+Run irb to start interactive Ruby CLI.  
+
```

Comment:
- [OPTIONAL] It will be nice to outline explicit and complete steps on how to clone and run your app locally as this will be of help to non technical users.